src/main.spec.ts

```diff
@@ -0,0 +1,17 @@
+import * as lib from "./main";
+describe("the interface of the library", () => {
+  it("should have at least these members otherwise it is definitely a breaking change", () => {
```

Comment:
Good idea to add something like this.

One remark: If the lib had additional members this is not a breaking change, but this test would fail. So perhaps you would want to use [`expect.objectContaining(object)`](https://jestjs.io/docs/expect#expectobjectcontainingobject) instead?